lerna.json

```diff
@@ -1,5 +1,19 @@
 {
+  "$schema": "https://json.schemastore.org/lerna",
+  "changelogPreset": "conventional-changelog-conventionalcommits",
+  "command": {
+    "publish": {
+      "message": "chore: publish"
+    },
+    "version": {
+      "allowBranch": "main",
+      "conventionalCommits": true,
+      "createRelease": "github"
+    }
+  },
+  "ignoreChanges": ["**/test/**", "**/*.md"],
   "npmClient": "yarn",
```

Comment:
```suggestion
  "npmClient": "npm",
```

Reply:
we should commit this some point right?

if not here then in
- https://github.com/LavaMoat/LavaMoat/pull/629

> so that lerna can adjust some of its internal logic when resolving configuration and packages

Reply:
yeah, good catch.